[Forome-982] Actualize e2e tests 2

viraxslot opened the pull request:
- actualized tests
- fixed selectors
- API endpoints moved to constants
- small refactoring

Commit:
fix(FOROME-982): updated second cypress spec

Commit:
Merge branch 'develop' of github.com:ForomePlatform/Anfisa-React-Client into FOROME-982-actualize-tests-2

Commit:
fix(FOROME-982): third cypress spec

Commit:
fix(FOROME-982): updated cypress, made fixes in tests

Commit:
fix(FOROME-982): updated cypress, made fixes in tests

Commit:
Merge branch 'develop' of github.com:ForomePlatform/Anfisa-React-Client into FOROME-982-actualize-tests-2

Commit:
chore(FOROME-982): merged develop branch

Commit:
fix(FOROME-982): selectors

Commit:
chore(FOROME-982): merged develop

Commit:
chore(FOROME-982): skip specs, fix selector

Lapk1n commented on src/pages/filter/common/view-variants/variant-content/variant-content.tsx:
```diff
@@ -7,6 +7,7 @@ import { Icon } from '@ui/icon'
 import { Loader } from '@ui/loader'
 import { VariantAspectsLayoutGallery } from '@components/variant-aspects-layout'
 import { TAspectDescriptor } from '@service-providers/dataset-level'
+import { ReturnedVariantsDataCy } from '../../../../../components/data-testid/returned-variants'
```
could be shorter {...} from '@components/data-testid...'

viraxslot replied:
Fixed using @AlMaQntr suggestion

Lapk1n commented on src/components/units-list/units-list-group/units-list-group.tsx:
```diff
@@ -7,6 +7,7 @@ import { TUnitGroup } from '@store/stat-units'
 import { DropdownArrow } from '@ui/dropdown-arrow'
 import { PredictionPowerIndicator } from '@components/prediction-power-indicator'
 import { TPropertyStatus } from '@service-providers/common'
+import { DecisionTreesResultsDataCy } from '../../data-testid/decision-tree-results.cy'
```
the same {...} from '@components/data-testid...'

Lapk1n commented on src/components/units-list/units-list-controls/units-list-controls.tsx:
```diff
@@ -6,6 +6,7 @@ import cn from 'classnames'
 import { t } from '@i18n'
 import { Icon } from '@ui/icon'
 import { InputSearch } from '@components/input-search'
+import { DecisionTreesResultsDataCy } from '../../data-testid/decision-tree-results.cy'
```
the same {...} from '@components/data-testid...'

Lapk1n commented on src/components/solution-control/solution-control-popover/solution-control-popover.tsx:
```diff
@@ -9,6 +9,7 @@ import { Popover } from '@ui/popover'
 import { IPopoverBaseProps } from '@ui/popover/popover.interface'
 import { popoverOffset } from '@pages/ws/ws.constants'
 import { ISolutionEntryDescription } from '@service-providers/common'
+import { DecisionTreesMenuDataCy } from '../../data-testid/decision-tree-menu.cy'
```
the same {...} from '@components/data-testid...'

Lapk1n commented on src/components/solution-control/solution-control-button/solution-control-button.tsx:
```diff
@@ -6,6 +6,7 @@ import cn from 'classnames'
 import { t } from '@i18n'
 import { Icon } from '@ui/icon'
 import { Loader } from '@ui/loader'
+import { DecisionTreesMenuDataCy } from '../../data-testid/decision-tree-menu.cy'
```
the same {...} from '@components/data-testid...'

AlMaQntr reviewed (commented):
I think, we should move `data-testid` dir outside of `src` dir.
Next, create `index.ts` inside it, where exports all identifiers.
And next, create an alias to `data-testid` dir e.g. `@data-testid`
As a result, we will have all test id imports like this:
```
import { SomeId, SomeOtherId } from '@data-testid'
```

Commit:
Merge branch 'develop' of github.com:ForomePlatform/Anfisa-React-Client into FOROME-982-actualize-tests-2

Commit:
chore(FOROME-982): updated imports

Commit:
chore(FOROME-982): updated imports

Commit:
chore(FOROME-982): removed index postfix

Commit:
fix(FOROME-982): cypress imports

viraxslot commented:
> I think, we should move `data-testid` dir outside of `src` dir.

@AlMaQntr Craco doesn't allow to import something outside the `src` folder so I moved `data-testid` folder to `src` and added an alias and `index.ts` with all exports.

Commit:
chore(FOROME-982): merged develop branch
